coreutils/Makefile

```diff
@@ -302,6 +302,10 @@ ${COREUTILS_NOLINK_DIR}:
 	if [ ! -z "$$GCDA_FILES" ] ; then \
 		rm -f $$GCDA_FILES ; \
 	fi
+	# Generate line coverage information
```

Comment:
@rasoolmaghareh This is a coverage computation for scalability programs using scalabilitycov.sh script. It is not applicable to coreutils. The coverage computation for coreutils is at Lines 300-304. Those are the ones that need to be enclosed within `if [ "${ENABLE_COVERAGE}" != "OFF" ] ; then \ .. \ fi`

Line 379-382 of this file also needs to be enclosed in the same if conditional, so that line coverage is excluded from the output experiment.csv spreadsheet. Also, Line 336 has to be changed so that it does not include the "llvm-cov line coverage" field in case coverage computation is disabled.

The way we generate small-coreutils.csv in Line 408 using cut needs change. cut needs to be given different field numbers in case coverage is disabled because we the field for line coverage is not there.